src/api/java/appeng/api/implementations/tiles/IConfigurableFluidInventory.java

```diff
@@ -0,0 +1,10 @@
+package appeng.api.implementations.tiles;
+
+import net.minecraftforge.fluids.capability.IFluidHandler;
+
+public interface IConfigurableFluidInventory {
```

Comment:
This should not be public API interface currently. There are still things which can be handled better in general and I would like to avoid having to do an API break just for that. Especially as it then still relies on an internal interface to actually work.

Reply:
I moved it to appeng.fluids.helper.IConfigurableFluidInventory